corehq/apps/locations/bulk_management.py

```diff
@@ -326,10 +346,11 @@ def validate_and_parse_stubs_from_excel(self):
 
         # all locations sheets should have correct headers
         location_stubs = []
+        optional_headers = [LOCATION_SHEET_HEADERS['custom_data'], LOCATION_SHEET_HEADERS['uncategorized_data']]
         for sheet_name, sheet_reader in sheets_by_title.items():
             if sheet_name != self.types_sheet_title:
-                actual = set(sheet_reader.fieldnames)
-                expected = set(LOCATION_SHEET_HEADERS.values())
+                actual = set(sheet_reader.fieldnames) - set(optional_headers)
+                expected = set(LOCATION_SHEET_HEADERS.values()) - set(optional_headers)
```

Comment:
Cool, so if they don't include a column for a required field, then it'll fail in a different place?